test/units/modules/cloud/amazon/test_cloudformation.py

```diff
@@ -73,7 +73,7 @@ def test_invalid_template_json(placeboify):
     assert "ValidationError" in m.exit_kwargs['msg']
 
 
-def test_basic_s3_stack(maybe_sleep, placeboify):
+def test_client_request_token_s3_stack(maybe_sleep, placeboify):
```

Comment:
Since this is now 2 tests, can you rename the stack being created in one of them? That way it'll be easier to distinguish the two sets of placebo recordings and avoid cross-contamination between the tests, just in case the delete at the end fails.

Reply:
Done, thanks for your feedback.